Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity.yml

```diff
@@ -762,17 +774,857 @@ script:
     execution: false
     name: ms-graph-security-auth-reset
     arguments: []
+  - deprecated: false
+    arguments:
+    - description: Which permissions the integration should have.
+      name: permission_type
+      isArray: true
+      defaultValue: all
+      auto: PREDEFINED
+      predefined:
+      - 'ediscovery'
+      - 'alerts'
+      - 'all'
```

Comment:
so the default is `all` so it should still be painless 
I was thinking that really ediscovery and graph security alerts arent really the same usecase. It can be a user wants to install this pack for alerts, doesnt at all care about ediscovery. I want him to be able to check his integration without failing on permission errors that he doesnt really need for his usecase

Im also assuming we will add a lot more purview commands here, which is really a collection of many different use cases. So I dont think its a good expectation that if you install this integration your user has commands for **all** of the usecases